sections/title.py

```diff
@@ -20,37 +20,17 @@ def heading():
             html.Div(
                 [
                     html.Span(
-                        'Effective',
-                        className = 'effective',
-                    ),
-                    html.Span(
-                        'Altruism',
-                        className = 'altruism',
-                    ),
-                ],
-                className='effective-altruism',
-            ),
-            html.Div(
-                [
-                    html.Span(
-                        'Data',
-                        className = 'data',
-                    ),
-                    html.Span(
-                        '.com',
-                        className = 'dot-com',
-                    ),
-                ],
-                className='data-dot-com',
-            ),
+                        'Effective Altruism Data'
```

Comment:
I know the vertical alignment is bad right now, but I don't want to lose the different fonts. Leave the original for now and I'll fix it.